src/main/java/org/broadinstitute/hellbender/tools/gvs/extract/ExtractCohort.java

```diff
@@ -143,25 +140,29 @@
     protected void onStartup() {
         super.onStartup();
 
-        if ( (filterSetInfoTableName != null || filterSetSiteTableName != null) && (filterSetName == null || filterSetName.equals(""))) {
-            throw new UserException("--filter-set-name must be specified if any filtering related operations are requested");
-        }
-
         Set<VCFHeaderLine> extraHeaderLines = new HashSet<>();
-        if (filterSetInfoTableName != null) {
-            FilterSensitivityTools.validateFilteringCutoffs(truthSensitivitySNPThreshold, truthSensitivityINDELThreshold, vqsLodSNPThreshold, vqsLodINDELThreshold, tranchesTableName);
-            Map<String, Map<Double, Double>> trancheMaps = FilterSensitivityTools.getTrancheMaps(filterSetName, tranchesTableName, projectID);
-
-            if (vqsLodSNPThreshold != null) { // we already have vqslod thresholds directly
-                extraHeaderLines.add(FilterSensitivityTools.getVqsLodHeader(vqsLodSNPThreshold, GATKVCFConstants.SNP));
-                extraHeaderLines.add(FilterSensitivityTools.getVqsLodHeader(vqsLodINDELThreshold, GATKVCFConstants.INDEL));
-            } else { // using sensitivity threshold inputs; need to convert these to vqslod thresholds
-                vqsLodSNPThreshold = FilterSensitivityTools.getVqslodThreshold(trancheMaps.get(GATKVCFConstants.SNP), truthSensitivitySNPThreshold, GATKVCFConstants.SNP);
-                vqsLodINDELThreshold = FilterSensitivityTools.getVqslodThreshold(trancheMaps.get(GATKVCFConstants.INDEL), truthSensitivityINDELThreshold, GATKVCFConstants.INDEL);
-                // set headers
-                extraHeaderLines.add(FilterSensitivityTools.getTruthSensitivityHeader(truthSensitivitySNPThreshold, vqsLodSNPThreshold, GATKVCFConstants.SNP));
-                extraHeaderLines.add(FilterSensitivityTools.getTruthSensitivityHeader(truthSensitivityINDELThreshold, vqsLodINDELThreshold, GATKVCFConstants.INDEL));
-            }
+
+        if (filterSetName == null || filterSetName.equals("")) {
+          // Should the below list be expanded?
```

Comment:
I would also like to add I would also like to add performGenotypeVQSLODFiltering = false (or some alternative) here. This should explicitly make sure we dont run into this issue if there is no needed filtering.